archive/dto.ksy

```diff
@@ -0,0 +1,57 @@
+meta:
+  id: dto
+  title: DTB/DTBO
+  license: CC0-1.0
+  file-extension: img
+  endian: be
+
+doc-ref:
+  - https://source.android.com/devices/architecture/dto/partitions
+  - https://android.googlesource.com/platform/system/libufdt/+/refs/heads/master/utils/
+
+types:
+  header:
+    seq:
+    - id: magic
+      contents: [ 0xd7, 0xb7, 0xab, 0x1e ]
+    - id: total_size
+      type: u4
+    - id: header_size
+      type: u4
+    - id: dt_entry_size
+      type: u4
+    - id: dt_entry_count
+      type: u4
```

Comment:
`num_entries` is preferred, see the [KSY Style Guide](https://doc.kaitai.io/ksy_style_guide.html#attr-id). If you want to keep a reference to the original name, use [`-orig-id`](https://doc.kaitai.io/user_guide.html#orig-id).

Reply:
Actually, I can't agree with you here. Names are known, they are fixed in documentation and they are "compatible"  with id naming restrictions. https://source.android.com/devices/architecture/dto/partitions I tried to replace them according to the style guide and adding -orig-id and it looks bad. Changing the fields names only introduces mess here. But I agree with you that I need to add fields-level docs.